airflow/contrib/operators/gcp_video_intelligence_operator.py

```diff
@@ -0,0 +1,245 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from google.protobuf.json_format import MessageToDict
+
+from airflow.contrib.hooks.gcp_video_intelligence_hook import CloudVideoIntelligenceHook
+from airflow.models import BaseOperator
+from google.cloud.videointelligence_v1 import enums
+
+
+class CloudVideoIntelligenceDetectVideoLabelsOperator(BaseOperator):
+    """
+    Performs video annotation, annotating video labels.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:CloudVideoIntelligenceDetectVideoLabelsOperator`.
+
+    :param input_uri: Input video location. Currently, only Google Cloud Storage URIs are supported,
+        which must be specified in the following format: gs://bucket-id/object-id.
+    :type input_uri: str
+    :param input_content: The video data bytes.
+        If unset, the input video(s) should be specified via input_uri. If set, input_uri should be unset.
+    :type input_content: bytes
+    :param output_uri: Optional, location where the output (in JSON format) should be stored. Currently, only
+        Google Cloud Storage URIs are supported, which must be specified in the following format:
+        gs://bucket-id/object-id.
+    :type output_uri: str
+    :param video_context: Optional, Additional video context and/or feature-specific parameters. See more:
+        https://googleapis.github.io/google-cloud-python/latest/videointelligence/gapic/v1/types.html#google.cloud.videointelligence_v1.types.VideoContext
```

Comment:
We have automatic linking to external documentation. Can you delete it because it will look bad?

Reply:
Deleted